services/apps/src/apps/containers.clj

```diff
@@ -91,7 +91,9 @@
   (let [tag  (get-tag image-map)
         name (:name image-map)
         url  (:url image-map)]
-    (when-not (image? image-map)
+    (if (image? image-map)
+      (first (select container-images
+                     (where {:name name :tag tag})))
```

Comment:
I'm guessing this change is for getting some tests to work? It doesn't seem permissions-related but neither does it seem necessarily wrong. What's going on here?